cves/CVE-2011-0722.yml

```diff
@@ -359,8 +421,8 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: 119
```

Comment:
119 is what I would say as well.

Reply:
Yes, fortunately NIST suggested an appropriate CWE here (unlike my other CVE).